src/main/kotlin/exercise2/task5/CreateUserDSL.kt

```diff
@@ -49,11 +49,16 @@ internal data class Address(
  */
 
 internal fun user(initUser: User.() -> Unit): User {
-    TODO("Implement me!!!")
+    val newUser = User()
+    newUser.initUser()
+    return newUser
 }
 
 internal fun User.address(initAddress: Address.() -> Unit): User {
-    TODO("Implement me!!!")
+    this.address = Address()
```

Comment:
Isto kao i komentar iznad:
```suggestion
    internal fun User.address(initAddress: Address.() -> Unit): User {
       return apply { this.address = Address.apply(initAddress) }
    }
```